docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -90,6 +90,9 @@ For `ipex-llm` NPU support, set the following environment variable with active `
 
 ```cmd
 set BIGDL_USE_NPU=1
```

Comment:
Maybe we can rename to `IPEX_LLM_USE_NPU` later ? Now it's a little strange with other env vars.

Reply:
> ?

Good catch! I think we could also verify whether it's still necessary later : )

Reply:
> Maybe we can rename to `IPEX_LLM_USE_NPU` later ? Now it's a little strange with other env vars.

Good catch! I think we could also verify whether this is still necessary later : )